README.md

```diff
@@ -32,8 +32,11 @@ Please add suggestions and emoji preferences to the [list of proposed topics](ht
 | [02/10/23](#021023) | Reinforcement Learning Human Feedback (RLHF) ([Slides](https://github.com/alan-turing-institute/transformers-reading-group/blob/main/sessions/12-rlhf/2023-10-02-rlhf.pdf)) | David Blackwell | [Eseoghene Ben-Iwhiwhu](https://github.com/dlpbc) |
 | [16/10/23](#161023) | Prompt Engineering ([Slides](https://github.com/alan-turing-institute/transformers-reading-group/blob/main/sessions/13-prompt-engineering/prompts.pdf)) | David Blackwell | [Martin Stoffel](https://github.com/mastoffel) |
 | [30/10/23](#301023) | Knowledge retrieval LLMs | David Blackwell | [Praveen Selvaraj](https://github.com/pravsels) |
+| [06/11/23](#061123) | Evaluating factuality in LLMs | David Blackwell | [Fede Nanni](https://github.com/fedenanni), [Ryan Chan](https://github.com/rchan26) |
```

Comment:
We could mark that this and the next are discussions, maybe starting the title with Discussion: ?